set.mm

```diff
@@ -715409,6 +715439,32 @@ contain the empty set ( ` -. (/) e. S ` ), no additional assumption is
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Logical disjunction and conjunction - extension
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( Rearrangement of 4 conjuncts: second and forth positions interchanged.
+     (Contributed by AV, 18-Feb-2022.) $)
```

Comment:
Did you try to minimize the theorems using ancom or an43 with it ? It may "pay for itself" and be moved to Main.

Reply:
No, I did not try to minimize prrofs in general by these theorems.  I have proved these theorems to shorten a special proof, but the result was a longer proof! Therefore these theorems are not used yet. But I wanted to keep them (for a future use).